apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -192,7 +253,10 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
         <Button
           text={isNewSection ? i18n.finishCreatingSections() : i18n.save()}
           color="purple"
-          onClick={e => saveSection(e, sections[0])}
+          onClick={e => {
+            recordSectionSetupEvent(e, sections[0]),
+              saveSection(e, sections[0], isNewSection);
```

Comment:
What's this comma syntax, is that intentional?

Reply:
But I updated it:)